lib/phoenix_live_view/html_algebra.ex

```diff
@@ -273,6 +273,10 @@ defmodule Phoenix.LiveView.HTMLAlgebra do
     {:inline, concat(["<%!--", text, "--%>"])}
   end
 
+  defp to_algebra({:eex, text, %{opt: opt}}, %{mode: :preserve}) do
```

Comment:
We were losing the spaces from the binary because it was formatting the eex expression which was messing up with our algorithm to count spaces.

https://github.com/phoenixframework/phoenix_live_view/blob/0e9d1107738c62c8bf33d4841e040017cbb329f4/lib/phoenix_live_view/html_algebra.ex#L156-L170

I believe we want to ignore it for this case and that fixes the issue.